### PR TITLE
Brians feature branch

### PR DESCRIPTION
**Pull request by bduimstra**

### CODE REVIEW

**Comment by macdhollister** [comment]
No. Your code is bad and you should feel bad.

**Comment by bduimstra** [comment]
IT’S TRUE :poop:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<html>
```
Comment (bduimstra) [inline index.html:1]: WHAT IS THIS GARBAGE

**Review by macdhollister — APPROVED** [review]
ok fine I guess it's good enough for my extremely high standards

**Pull request merged**